### ISSUE TITLE
How does auto updating work?

### ISSUE THREAD

**Issue by fagitomania** [Kyusung4698/PoE-Overlay#307]
I have to click the link and redownload manually? I cant seem to find how it works automatically

**Comment by Kyusung4698** [comment]
There was a bug with auto update prior 0.6.2. It should work now like this:

- check every 5 minutes for new updates
- once a new update is found it'll download the new version (unless disabled in menu)
- once the download is complete - it'll ask you to restart the app. 
- once restarted the app should be update

**Comment by Kyusung4698** [comment]
I'll close this issue. If this still seems unclear. Feel free to reopen.

**Comment by ZeroCool940711** [comment]
@Kyusung4698 I think it would be nice to have a `Check for Update` button on the UI to manually force an update check and have a progress bar or something showing how the download is going, I know the app automatically checks for update every 5 minutes but sometimes we want to force an update check before that and for what I see there is no way right now to force an update unless you download the new setup yourself from the Github release page, so, if you are playing and suddently you recieve the email notification that there is a new update you have to wait 5 minutes for the app to update itself or you have to download manually the setup file and install it yourself. 

Btw, Im commenting here instead of opening a new Issue about the Auto Update thing as I think it will be better
